test/shell/exec-usergroup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+set -x
+
+if ! timeout 1s sudo -A true; then
+	echo "sudo disabled: not checking exec user and group"
+	exit
+fi
+
+function setup {
+	mkdir -p /tmp/mgmt/test-exec-usergroup/
+	sudo -A chown nobody:$group /tmp/mgmt/test-exec-usergroup/
+	sudo -A chmod 775 /tmp/mgmt/test-exec-usergroup/
+}
+
+function cleanup {
+	sudo -A rm -rf /tmp/mgmt/test-exec-usergroup
+}
+
+# run_test will run each test. It takes 3 parameters:
+# - $1: graph (e.g. exec-usergroup-nobody.yaml)
+# - $2: user to be tested (e.g. nobody or "")
+# - $3: group to be tested (e.g. nobody or "")
+function run_test() {
+	graph=$1
+	user=$2
+	group=$3
+
+	setup
+
+	# run till completion
+	timeout --kill-after=30s 25s sudo -A ./mgmt run --yaml ./exec-usergroup/$graph --converged-timeout=5 --no-watch --tmp-prefix &
```

Comment:
path's like this should be in a $VARIABLE at the top of the file, eg:

BASE_PATH="/tmp/mgmt/"
BASE_PATH_TEST="${BASE_PATH}exec-usergroup"

and the $graph should be `${graph} for variables being expanded